Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
Presumably because it is not portable, ie if you change the compiler it won't work. Doing it on a per file basis controlled by the makefile is much more portable.